**На сколько $\sigma$ (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 см$^2$?**  

Воспользуемся формулой расчета $Z$-оценки:  
$\displaystyle Z=\frac{X-\bar x}{\sigma}=\frac{190-178}{\sqrt{25}}=\frac{12}{5}$ , где:  
    $\bar x$ = M(X) = 178  
    $\sigma$ = $\sqrt{D(X)}$ = $\sqrt{25}$

In [11]:
print(f'Рост человека, равный 190 см, отклоняется от математического ожидания роста в популяции, \n'
      f'в которой M(X) = 178 см и D(X) = 25 см\u00b2 на {(190-178)/25**(1/2):.1f}σ')

Рост человека, равный 190 см, отклоняется от математического ожидания роста в популяции, 
в которой M(X) = 178 см и D(X) = 25 см² на 2.4σ


---  
Ответ:  
Рост человека, равный 190 см, отклоняется от математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 см$^2$на $2.4\sigma$  

---